### PR TITLE
Fix `_latex_` exponents

### PR DESCRIPTION
**Pull request by grhkm21**
Fix `_latex_` exponents in some algebraic geometry code.

### CODE REVIEW

**Comment by mkoeppe** [comment]
Test failures in CI

**Comment by grhkm21** [comment]
all good now, CI passes

File: src/sage/schemes/affine/affine_space.py [hunk 1]
```diff
@@ -332,14 +332,14 @@ def _latex_(self):
...
 
             sage: print(latex(AffineSpace(1, ZZ, 'x')))
-            \mathbf{A}_{\Bold{Z}}^1
+            \mathbf{A}_{\Bold{Z}}^{1}
 
         TESTS::
 
-            sage: AffineSpace(3, Zp(5), 'y')._latex_()                                  # needs sage.rings.padics
-            '\\mathbf{A}_{\\Bold{Z}_{5}}^3'
+            sage: AffineSpace(11, Zp(5), 'y')._latex_()                                  # needs sage.rings.padics
```
Comment (mkoeppe) [inline src/sage/schemes/affine/affine_space.py]:
```suggestion
            sage: AffineSpace(11, Zp(5), 'y')._latex_()                                 # needs sage.rings.padics
```

File: src/sage/schemes/projective/projective_space.py [hunk 2]
```diff
@@ -590,15 +590,14 @@ def _latex_(self):
...
 
             sage: print(latex(ProjectiveSpace(1, ZZ, 'x')))
-            {\mathbf P}_{\Bold{Z}}^1
+            {\mathbf P}_{\Bold{Z}}^{1}
 
         TESTS::
 
-            sage: ProjectiveSpace(3, Zp(5), 'y')._latex_()                              # needs sage.rings.padics
-            '{\\mathbf P}_{\\Bold{Z}_{5}}^3'
+            sage: ProjectiveSpace(11, Zp(5), 'y')._latex_()                              # needs sage.rings.padics
```
Comment (mkoeppe) [inline src/sage/schemes/projective/projective_space.py]:
```suggestion
            sage: ProjectiveSpace(11, Zp(5), 'y')._latex_()                             # needs sage.rings.padics
```

**Review by mkoeppe — APPROVED** [review]
Otherwise LGTM

**Pull request merged**